packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -621,13 +620,24 @@ contract OPContractsManagerGameTypeAdder is OPContractsManagerBase {
                     revert OPContractsManagerGameTypeAdder_MixedGameTypes();
                 }
 
```

Comment:
this code block is slightly cut off in the github diff but it:
```solidity
if (hasLegacyGame && hasSuperGame) {
    revert OPContractsManagerGameTypeAdder_MixedGameTypes();
}
```
are there any other combinations that should not be allowed at this time, for example we are ok with permissioned cannon AND cannon+kona AND no cannon?